lib/mixin/vpp.py

```diff
@@ -224,3 +224,15 @@ def check_color_props(self):
     metrics2.check(**params)
 
   check_range  = check_color_props
+
+  def check_drawbox(self):
+    owidth, oheight = self.width, self.height
```

Comment:
Assignment not needed.  Just pass directly to the `filesize` check

Reply:
Thanks for pointing it out, updated.